package.json

```diff
@@ -91,48 +91,48 @@
   },
   "dependencies": {
     "@opentelemetry/api": "^1.0.3",
-    "@opentelemetry/context-async-hooks": "~0.25.0",
-    "@opentelemetry/core": "~0.25.0",
+    "@opentelemetry/context-async-hooks": "~1.0.0",
+    "@opentelemetry/core": "~1.0.0",
     "@opentelemetry/exporter-collector-grpc": "^0.25.0",
-    "@opentelemetry/exporter-jaeger": "~0.25.0",
+    "@opentelemetry/exporter-jaeger": "~1.0.0",
     "@opentelemetry/instrumentation": "~0.25.0",
-    "@opentelemetry/propagator-b3": "~0.25.0",
-    "@opentelemetry/resources": "~0.25.0",
-    "@opentelemetry/sdk-trace-base": "~0.25.0",
-    "@opentelemetry/sdk-trace-node": "~0.25.0",
-    "@opentelemetry/semantic-conventions": "~0.25.0",
+    "@opentelemetry/propagator-b3": "~1.0.0",
+    "@opentelemetry/resources": "~1.0.0",
+    "@opentelemetry/sdk-trace-base": "~1.0.0",
+    "@opentelemetry/sdk-trace-node": "~1.0.0",
+    "@opentelemetry/semantic-conventions": "~1.0.0",
     "@types/sonic-boom": "^0.7.0",
     "jaeger-client": "^3.18.1",
     "semver": "^7.3.5"
   },
   "peerDependencies": {
-    "@opentelemetry/instrumentation-bunyan": ">=0.24.0 <1",
-    "@opentelemetry/instrumentation-cassandra-driver": ">=0.23.0 <1",
-    "@opentelemetry/instrumentation-dns": ">=0.23.0 <1",
-    "@opentelemetry/instrumentation-express": ">=0.23.0 <1",
-    "@opentelemetry/instrumentation-graphql": ">=0.23.0 <1",
-    "@opentelemetry/instrumentation-grpc": ">=0.23.0 <1",
-    "@opentelemetry/instrumentation-hapi": ">=0.23.0 <1",
-    "@opentelemetry/instrumentation-http": ">=0.23.0 <1",
-    "@opentelemetry/instrumentation-ioredis": ">=0.23.0 <1",
-    "@opentelemetry/instrumentation-knex": ">=0.23.0 <1",
-    "@opentelemetry/instrumentation-koa": ">=0.23.0 <1",
-    "@opentelemetry/instrumentation-memcached": ">=0.23.0 <1",
-    "@opentelemetry/instrumentation-mongodb": ">=0.23.0 <1",
-    "@opentelemetry/instrumentation-mysql": ">=0.23.0 <1",
-    "@opentelemetry/instrumentation-mysql2": ">=0.23.0 <1",
-    "@opentelemetry/instrumentation-net": ">=0.23.0 <1",
-    "@opentelemetry/instrumentation-pg": ">=0.23.0 <1",
-    "@opentelemetry/instrumentation-pino": ">=0.23.0 <1",
-    "@opentelemetry/instrumentation-redis": ">=0.23.0 <1",
-    "@opentelemetry/instrumentation-winston": ">=0.24.0 <1",
-    "opentelemetry-instrumentation-amqplib": ">=0.23.0 <1",
-    "opentelemetry-instrumentation-aws-sdk": ">=0.23.0 <1",
-    "opentelemetry-instrumentation-elasticsearch": ">=0.23.0 <1",
-    "opentelemetry-instrumentation-kafkajs": ">=0.23.0 <1",
-    "opentelemetry-instrumentation-mongoose": ">=0.23.0 <1",
-    "opentelemetry-instrumentation-sequelize": ">=0.23.0 <1",
-    "opentelemetry-instrumentation-typeorm": ">=0.23.0 <1"
+    "@opentelemetry/instrumentation-bunyan": ">=0.25.0 <1",
```

Comment:
I think this is fine, but shouldn't `@opentelemetry/instrumentation` have the same supported versions as `@opentelemetry/instrumentation-*`?